aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1866,6 +1867,38 @@ impl VMValidator for AptosVM {
             }
         }
 
+        if !self
+            .vm_impl
+            .get_features()
+            .is_enabled(FeatureFlag::WEBAUTHN_SIGNATURE)
+        {
+            let is_webauthn_signature = |signature: &AnySignature| -> bool {
+                if let AnySignature::WebAuthn { .. } = signature {
+                    return true;
+                }
+                false
+            };
```

Comment:
Can you make this inline somewhere for perf?

Reply:
I think i'd have to make it a standalone function right?